### PR TITLE
[infra] Use coverage utils from Chromium and other fixes (follow-up #1547).

### PR DESCRIPTION
**Pull request by Dor1s**

### CODE REVIEW

**Comment by Dor1s** [comment]
I need to land https://chromium-review.googlesource.com/c/chromium/src/+/1183695 first to double check this locally end-to-end.

File: infra/base-images/base-runner/Dockerfile [hunk 1]
```diff
@@ -17,8 +17,10 @@
 FROM gcr.io/oss-fuzz-base/base-image
 MAINTAINER mike.aizatsky@gmail.com
 RUN apt-get install -y zip file libunwind8 binutils libblocksruntime0 \
-    fonts-dejavu python3 libcap2 wget
-COPY bad_build_check coverage coverage_helper.py download_corpus llvm-cov \
+    fonts-dejavu python3 python3-pip libcap2 wget git
```
Comment (inferno-chromium) [inline infra/base-images/base-runner/Dockerfile]: alpha order

File: infra/base-images/base-runner/coverage [hunk 2]
```diff
@@ -127,9 +129,13 @@ llvm-cov show -format=html -output-dir=$REPORT_DIR \
 # Export coverage summary in JSON format.
 llvm-cov export -summary-only $LLVM_COV_ARGS > $SUMMARY_FILE
 
+# Post process HTML report.
+coverage_helper -v post_process -output-dir=$REPORT_DIR -src-root-dir=$OUT \
```
Comment (inferno-chromium) [inline infra/base-images/base-runner/coverage]: nit: i think -src-root-dir looks better first, followed by output ones like -output-dir -summary-file

File: infra/base-images/base-runner/Dockerfile (hunk 1 above)
Comment (Dor1s) [reply infra/base-images/base-runner/Dockerfile]: Done. Can do the same for other files (probably a separate CL would be better).

File: infra/base-images/base-runner/coverage (hunk 2 above)
Comment (Dor1s) [reply infra/base-images/base-runner/coverage]: Yeah, good point, src root and summary are inputs, so I'll pass them first. Thanks!

**Pull request merged**